src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.ts

```diff
@@ -222,4 +223,8 @@ export class GqrRqgViewResponsesComponent extends InstructorResponsesViewBase im
       }
     }
   }
+
+  removeAnonymousHash(str: string) : string {
+    return StringHelper.removeAnonymousHash(str);
+  }
```

Comment:
I think instead of creating a method that formats the string to display, you can create a (pure) pipe that removes the anonymous hash and use it directly in the html file. Pipes are better since it will only run when the underlying value changes and they can be declared once and used throughout the app.